src/test/addrman_tests.cpp

```diff
@@ -1043,13 +1017,39 @@ BOOST_AUTO_TEST_CASE(load_addrman)
     BOOST_CHECK(addrman2.size() == 3);
 }
 
+// Produce a corrupt peers.dat that claims 20 addrs when it only has one addr.
+static CDataStream MakeCorruptPeersDat()
+{
+    CDataStream s(SER_DISK, CLIENT_VERSION);
+    s << ::Params().MessageStart();
```

Comment:
nit, since we are already in global namespace, the scope-resolution operator could be dropped (like in `AddrmanToStream` above):
```suggestion
    s << Params().MessageStart();
```

Reply:
I'm not sure, but I think there's a preference to be more explicit when we're calling global functions like this.

Reply:
Okay, my assumption was that the scope-resolution operator is only preferred if there is the possibility of namespace ambiguity (e.g. in class methods). I guess it should then also be added to `Params()` in `AddrmanToStream` to be consistent. Again, not a big deal, but maybe something to include in potential follow-up PRs.